config/locales/de.yml

```diff
@@ -477,6 +480,8 @@ de:
         new_request_night_1: "Hallo %{first_name}, heute Nacht ist mir etwas zum Thema „%{request_title}“ eingefallen. Wenn du antworten möchtest, klicke auf 'Antworten'."
         new_request_night_2: "Hallo %{first_name}, ich weiß, es ist schon spät. Aber ich möchte gern mehr zum Thema „%{request_title}“ erfahren. Wenn du antworten möchtest, klicke auf 'Antworten'."
         new_request_night_3: "Hallo %{first_name}, zum Thema „%{request_title}“ möchte ich heute noch gern mehr erfahren. Wenn du antworten möchtest, klicke auf 'Antworten'."
+      direct_message_template:
+        new_direct_message: "Hallo %{first_name}, wir haben eine neue Nachricht für dich. Wenn du antworten möchtest, klicke auf 'Antworten'."
```

Comment:
According to @FrauCsu 
Hallo %{first_name}, wir haben eine neue Nachricht für dich. Wenn du sie öffnen möchtest, klicke 'Öffnen'.